forms-flow-web/src/components/Form/Stepper.js

```diff
@@ -376,45 +370,37 @@ class StepperPage extends PureComponent {
               <i className="fa fa-chevron-left fa-lg m-3" />
             </Link>
           ) : null}
-          <div className="paper-root">
-            <Grid
-              container
-              direction="row"
-              justify="flex-start"
-              alignItems="baseline"
-            >
-              <Grid item xs={12} spacing={3}>
-                <Stepper
-                  alternativeLabel
-                  nonLinear
-                  activeStep={this.state.activeStep}
-                >
-                  {steps.map((label, index) => {
-                    return (
-                      <Step key={index}>
-                        <StepLabel>{label}</StepLabel>
-                      </Step>
-                    );
-                  })}
-                </Stepper>
+          <Container fluid className="paper-root">
+            <Row>
+              <Col xs={12}>
```

Comment:
the better way is you can make the stepper as a component because we are using stepper in bundle also